From 99c75944a96c321dbf06d7126fab44f766ff06e5 Mon Sep 17 00:00:00 2001
From: Jeremy Bicha <jbicha@debian.org>
Date: Mon, 5 Oct 2020 20:23:51 -0400
Subject: [PATCH] metainfo: Don't try to access the internet when validating

It is common for network access to be unavailable during
a build. This should not fail the build.
---
 data/meson.build | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/data/meson.build b/data/meson.build
index 4c08bd0a..ccff2d80 100644
--- a/data/meson.build
+++ b/data/meson.build
@@ -42,7 +42,7 @@ appstream_util = find_program('appstream-util', required: false)
 if appstream_util.found()
   test(
       'validate-metainfo', appstream_util,
-      args: ['validate', metainfo_file.full_path()]
+      args: ['validate', '--nonet', metainfo_file.full_path()]
       )
 endif
 
-- 
GitLab

